auth_saml/models/saml_request.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (C) 2020 GlodoUK <https://glodo.uk/>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class SamlToken(models.TransientModel):
+    _name = "auth_saml.request"
```

Comment:
If the provider model is called `auth.saml.provider`

What about calling this one:

```suggestion
    _name = "auth.saml.request"
```